db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java

```diff
@@ -320,40 +320,22 @@ public void printSubNodes(int depth){
     public FromList getFromList(){ return fromList; }
 
     /**
-     * Find colName in the result columns and return underlying columnReference.
-     * Note that this function returns null if there are more than one FromTable
-     * for this SelectNode and the columnReference needs to be directly under
-     * the resultColumn. So having an expression under the resultSet would cause
-     * returning null.
-     *
-     * @param    colName        Name of the column
-     * @return ColumnReference    ColumnReference to the column, if found
+     * Remap the column reference node to underlying column reference in the select, if possible.
```

Comment:
Does the method really "remap" or simply gives a source column reference? The name suggests the latter.

Reply:
Thanks @jaceklaskowski for the review. "Remap" here I meant to "find the corresponding source column reference". I updated the comment to make the comment and function name consistent.